instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/OpenTelemetryAutoConfiguration.java

```diff
@@ -33,29 +33,37 @@
 @EnableConfigurationProperties(SamplerProperties.class)
 public class OpenTelemetryAutoConfiguration {
 
-  @Bean
-  @ConditionalOnMissingBean
-  public OpenTelemetry openTelemetry(
-      SamplerProperties samplerProperties,
-      ObjectProvider<ContextPropagators> propagatorsProvider,
-      ObjectProvider<List<SpanExporter>> spanExportersProvider) {
-    SdkTracerProviderBuilder tracerProviderBuilder = SdkTracerProvider.builder();
-
-    spanExportersProvider.getIfAvailable(Collections::emptyList).stream()
-        // todo SimpleSpanProcessor...is that really what we want here?
-        .map(SimpleSpanProcessor::create)
-        .forEach(tracerProviderBuilder::addSpanProcessor);
-
-    SdkTracerProvider tracerProvider =
-        tracerProviderBuilder
-            .setSampler(Sampler.traceIdRatioBased(samplerProperties.getProbability()))
-            .build();
-
-    ContextPropagators propagators = propagatorsProvider.getIfAvailable(ContextPropagators::noop);
-
-    return OpenTelemetrySdk.builder()
-        .setTracerProvider(tracerProvider)
-        .setPropagators(propagators)
-        .build();
+  @Configuration
+  @ConditionalOnMissingBean(OpenTelemetry.class)
+  public static class OpenTelemetryBeanConfig {
+
+    @Bean(destroyMethod = "close")
```

Comment:
FWIU, `close` will be called automatically without needing to set it here

Reply:
it looks like there's one other place with `destroyMethod = "close"`, going to merge and I'll send a separate PR to update both spots